bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/scheduler/ExpressionThreadPoolManager.java

```diff
@@ -38,8 +37,6 @@
 
     private final static Logger logger = LoggerFactory.getLogger(ExpressionThreadPoolManager.class);
```

Comment:
I feared this would catch sb's eye 😉 
No, it's not _that_ easy, but will do so.